### DISCUSSION TITLE
Quire Preview "caniuselite is outdated, run npx browserslist@latest --update-db" message

### DISCUSSION THREAD

**Discussion by anniemcewen** [thegetty/quire#288] [Get Help] [answered]
Hi all,

I am running into this message and I am not sure what to do:

loaner@OCIT’s MacBook Air quire-and-i-must-scream % quire preview
Launching preview server
ℹ Navigate to http://localhost:1313 to see your changes.
ℹ Press Control+C or type "quire stop" to stop the preview.
10% building 0/0 modules 0 active
webpack is watching the files…

**10% building 0/1 modules 1 active ...ds/quire-and-i-must-scream/themes/default/source/js/application.jsBrowserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating**

I have tried running the npx command, and it says success, but then it continues to happen and my preview won't update with any recent changes. 

I have also tried the classic "turn it off and on again" technique, and I have reinstalled the v14.18.1 release of Node.js to make sure that wasn't a problem. 

I am a total newbie at this (Art History Grad Student with limited tech knowledge), so any advice would be helpful. Thanks!

**Comment by Erin-Cecele** [comment, marked as answer]
Hi @anniemcewen, you can ignore the "caniuse-lite is outdated" message that appears in your terminal. The only terminal messages you need to worry about are any that say ERR or ERROR. 

What happens when you go to localhost:1313? Are you able to preview your project?

**Reply by anniemcewen** [reply]
Wait, I fixed it!! I was connecting to the wrong folder in terminal. Thanks for your help!

**Reply by Erin-Cecele** [reply]
@anniemcewen That's great news!! Let us know if you need assistance with anything else.

**Comment by anniemcewen** [comment]
Hi Erin! I am able to see the preview, but the changes I am making are not updating. For example, I set a new cover image with a new jpeg and I still see the old image, even after merging the change and syncing with Github.

From: Erin Cecele Dunigan ***@***.***>
Date: Thursday, April 21, 2022 at 2:33 PM
To: thegetty/quire ***@***.***>
Cc: McEwen, Annie ***@***.***>, Mention ***@***.***>
Subject: [External] Re: [thegetty/quire] Quire Preview "caniuselite is outdated, run npx ***@***.*** --update-db" message (Discussion #288)

Hi @anniemcewen<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fanniemcewen&data=05%7C01%7Cannie.mcewen%40emory.edu%7C61d218a4fe7a4c54ad7108da23c57c91%7Ce004fb9cb0a4424fbcd0322606d5df38%7C0%7C0%7C637861628369213338%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=kk7Lr4VCy%2Fkhdm0VdxVAJa23upoaiyCJlF%2B2IC%2FkmtU%3D&reserved=0>, you can ignore the "caniuse-lite is outdated" message that appears in your terminal. The only terminal messages you need to worry about are any that say ERR or ERROR.

What happens when you go to localhost:1313? Are you able to preview your project?
